wahlordnung.md

```diff
@@ -64,54 +64,60 @@ Diese Wahlordnung gilt für alle Wahlen zum Fachschaftsrat IT-Systems Engineerin
+(4) Wenn eine Person sowohl per E-Mail als auch per Wahlzettel abgestimmt hat, verfällt die Stimmabgabe aus der E-Mail.
 
-(5) Sofern ein Wähler sowohl per E-Mail als auch per Wahlzettel abgestimmt hat, verfällt die Stimmabgabe aus der E-Mail.
+(5) Bei mehreren Personen gleichen Namens wird für die Stimmabgabe der in der HPI-Domäne geführte Name mit nachgestellter Zahl verwendet. Der Wahlausschuss weist in der Wahlausschreibung und zur Eröffnung der Wahl darauf hin.
 
 
 ## § 10 Ergebnisfeststellung
 
-(1) Nach Abschluss der Wahl stellt der Wahlausschuss die Zahl der für jeden Kandidaten abgegebenen Stimmen fest. Die Stimmenauszählung erfolgt spätestens am dritten Vorlesungstag nach der Wahl. Sie erfolgt öffentlich für alle Mitglieder der Fachschaft und muss durch den Wahlausschuss spätestens zwei Tage vorher per E-Mail an die Fachschaft angekündigt werden.
+(1) Nach Abschluss der Wahl stellt der Wahlausschuss fest, für welche Person wie viele Stimmen abgegeben wurden. Die Stimmenauszählung erfolgt direkt im Anschluss an die Wahl oder am nächsten auf die Wahl folgenden Vorlesungstag. Sie erfolgt öffentlich für alle Mitglieder der Fachschaft und muss durch den Wahlausschuss spätestens zwei Tage vor der Wahl per E-Mail an die Fachschaft angekündigt werden.
 
-(2) In den Fachschaftsrat, die Studienkommission und den Prüfungsausschuss gewählt sind die Personen, die jeweils die meisten Stimmen auf sich vereinen. Sind weniger als die notwendige Anzahl Personen gewählt, sind Neuwahlen notwendig.
+(2) In den Fachschaftsrat, die Studienkommission und den Prüfungsausschuss gewählt sind die Personen, die jeweils die meisten Stimmen auf sich vereinen. Sind weniger als die notwendige Anzahl Personen gewählt, sind Neuwahlen für das jeweilige Gremium notwendig.
 
-(3) Bei Stimmengleichheit um den letzten Platz entstehen im Fachschaftsrat Überhangmandate. Bei Stimmengleichheit in den Wahlen für die studentischen Mitglieder in der Studienkommission und dem Prüfungsausschuss müssen sich die betroffenen Kandidaten mit gleicher Stimmenzahl zusammen mit dem Wahlausschuss auf ein Lösungsverfahren einigen.
+(3) Bei Stimmengleichheit um den letzten Platz entstehen im Fachschaftsrat Überhangmandate. Bei Stimmengleichheit in den Wahlen für die studentischen Mitglieder in der Studienkommission oder dem Prüfungsausschuss müssen sich die betroffenen Personen mit gleicher Stimmenzahl zusammen mit dem Wahlausschuss auf ein Lösungsverfahren einigen.
 
-(4) Jede gewählte Person muss dem Wahlausschuss innerhalb einer Woche mitteilen, ob er oder sie die Wahl annimmt. Ansonsten gilt die Annahme als verweigert.
+(4) Der Wahlausschuss informiert nach Abschluss der Auszählung umgehend alle gewählten Personen und fragt sie nach Annahme der Wahl.
+
+(5) Die Annahme muss dem Wahlausschuss innerhalb von zwei Vorlesungstagen mitgeteilt werden. Ansonsten gilt die Annahme als verweigert.
```

Comment:
innerhalb von zwei tagen inkl des tages der auszählung oder nicht?

in beiden fällen finde ich die frist sehr knapp. im zweifel sind es nur 48 stunden. das ist sehr wenig, wenn man z.B. sich gar nicht zur wahl gestellt hat und sich das noch überlegen möchte, oder wenn man mal für 48h keine mails beantworten will/kann, für das es vielerlei mögliche gründe gibt.